lib/SyTest/Synapse.pm

```diff
@@ -433,12 +433,13 @@ sub clear_db_pg
    my $self = shift;
    my %args = @_;
 
-   $self->{output}->diag( "Clearing Pg database $args{database} on $args{host}" );
+   my $host = $args{host} || '';
```

Comment:
Use the `//` operator as we care about definedness, not truth (just in case for some bizarre reason we want to use "0" as a host)